kubernetes/kubernetes
# [WIP] Fix NotFound errors do not line up with API endpoint's group version

Fixes https://github.com/kubernetes/kubernetes/issues/48959
manually test via kubectl
- [x] Create: kubectl create -f [nginx1.yaml](https://gist.githubusercontent.com/shiywang/58fee17c91d68b8d98c13ecbca16859a/raw/3d29f7962871f3f0acf68d5c5f9de0852ab8f78a/nginx1.yaml)  --v=5
- [x] Get: kubectl get deployments.apps foo  --v=5
- [x] Update: kubectl patch deployments.app  foo -p '{"spec":{"unschedulable":true}}' --v=5
- [ ] List: kubectl get deployments.apps  --v=5  **return "No resources found." how to trigger that by kubectl** ?
- [ ] Delete: kubectl delete deployments.apps foo  --v=5

I will also add some test soon, which kind of test should I add, @sttts still some intergration test ?

```release-note
NONE
```
